app/views/shared/eligible_region_content_components/_proof_of_qualifications.html.erb

```diff
@@ -0,0 +1,9 @@
+<p class="govuk-body">You must provide:</p>
+<ul class="govuk-list govuk-list--bullet">
+    <li>Your certificate from the qualification that led to you being a recognised teacher (this might be your university degree, or a separate qualification).</li>
+    <li>A transcript (or diploma supplement) for that qualification, issued by the awarding body. If your teaching qualification was part of your university degree, use your degree transcript.</li>
```

Comment:
```suggestion
  <li>Your certificate from the qualification that led to you being a recognised teacher (this might be your university degree, or a separate qualification).</li>
  <li>A transcript (or diploma supplement) for that qualification, issued by the awarding body. If your teaching qualification was part of your university degree, use your degree transcript.</li>
```